docs/hub/timm.md

```diff
@@ -0,0 +1,162 @@
+# Using Timm at Hugging Face
+
+`timm`, also known as [pytorch-image-models](https://github.com/rwightman/pytorch-image-models), is an open-source collection of state-of-the-art PyTorch image models, pretrained weights, and utility scripts for training, inference, and validation.
+
+## Exploring Timm in the Hub
+
+You can find a number of `timm` models by filtering at the left of the [models page](https://huggingface.co/models?library=timm&sort=downloads).
+
+All models on the Hub come up with useful feature:
+1. An automatically generated model card, which model authors can complete with [information about their model](./models-cards).
+2. Metadata tags that help for discoverability, allowing users to find the model when searching for `timm` models.
+3. An [interactive widget](./models-widgets) you can use to play out with the model directly in the browser
+4. An [Inference API](./models-inference) that allows users to make inference requests
+
+## Using existing models
+
+Any `timm` model from the Hugging Face Hub can be loaded with a single line of code as long as you have `timm` installed! Once you've selected a model from the Hub, pass the model's ID prefixed with `hf_hub:` to `timm`'s `create_model` method to download and instantiate the model. 
+
+```py
+import timm
+
+# Loading https://huggingface.co/sgugger/resnet50d
+model = timm.create_model("hf_hub:sgugger/resnet50d", pretrained=True)
+```
+
+If you want to see how to load a specific model, you can click **Use in timm** and you will be given a working snippet that you can load it! 
+
+<div class="flex justify-center">
```

Comment:
For some reason, when I go to the `sgugger/resnet50d` repo, it triggers my antivirus 🫣. Is this just me?

<img width="728" alt="Screen Shot 2022-09-02 at 2 29 01 PM" src="https://user-images.githubusercontent.com/59462357/188237865-452478ff-b985-4174-ba6c-7f470bbfef33.png">

Reply:
Oh weird! I guess your antivirus doesn't like neurohive.io. I'll change it to a different repo!